src/meshapi/admin/models/building.py

```diff
@@ -56,29 +59,31 @@ class Meta:
 
 
 @admin.register(Building)
-class BuildingAdmin(ImportExportModelAdmin, ExportActionMixin):
+class BuildingAdmin(RankedSearchMixin, ImportExportModelAdmin, ExportActionMixin):
     form = BuildingAdminForm
     list_display = ["__str__", "street_address", "primary_node"]
     search_fields = [
         # Sometimes they have an actual name
         "nodes__name__icontains",
         # Address info
         "street_address__icontains",
-        "city__icontains",
-        "state__icontains",
         "zip_code__iexact",
         "bin__iexact",
         # Search by NN
         "nodes__network_number__iexact",
         "installs__install_number__iexact",
-        # Search by Member info
-        "installs__member__name__icontains",
-        "installs__member__primary_email_address__icontains",
-        "installs__member__phone_number__iexact",
-        "installs__member__slack_handle__iexact",
         # Notes
-        "notes__icontains",
+        "@notes",
```

Comment:
What does this mean vs `notes__icontains`?

Reply:
The `@` symbol instructs Django to construct the queryset with `__search`, which uses [postgres' full-text search features](https://docs.djangoproject.com/en/5.1/ref/contrib/postgres/search/#the-search-lookup). Some of this is [over my head](https://www.postgresql.org/docs/current/datatype-textsearch.html#DATATYPE-TSVECTOR), but I think basically it does more fuzzy matching, and does some search-engine style splitting of the input so that the precise placement of punctuation, etc. doesn't totally blow up your results. As I understand it, `__icontains` is good for short strings where you're likely to want exact matches, but `__search` is better for long blocks of text where the user is unlikely to remember the exact details character-for-character